src/js/components/Drop/stories/Meter.js

```diff
@@ -1,34 +1,34 @@
 import React, { useEffect, useRef, useState } from 'react';
 
-import { Box, Drop, Grommet, Meter } from 'grommet';
-import { grommet } from 'grommet/themes';
+import { Box, Drop, Meter } from 'grommet';
 
 const TestDrop = () => {
   const targetRef = useRef();
 
   const [, setShowDrop] = useState(false);
   useEffect(() => setShowDrop(true), []);
   return (
-    <Grommet theme={grommet} full>
```

Comment:
@jcfilben just pointing out another case of `full`, as you mentioned it on another PR.

Reply:
Thanks for bringing this up, in this case the `full` prop isn't affecting the core behavior of the story we are trying to demonstrate/test. So I think it is fine if we don't have the `full` prop on this story.